### ISSUE TITLE
<a> anchor tags get transformed and broken

### ISSUE THREAD

**Issue by eedrah** [davidmfoley/node-trucker#21]
It seems that trucker is incorrectly transforming a `"/"` found inside an anchor tag. Note that this file is jsx, but I'm guessing there would be cases inside normal js files too?

Here's an example - the second transformation is correct, but the first one is not.

```
> trucker jsx/views/presentationLayer/Sidebars.js jsx/views/sidebars/
2 changes in 2 files.
jsx/views/presentationLayer/Sidebars.js  ->  jsx/views/sidebars/Sidebars.js
27: -             <a href="/">
27: +             <a href="../presentationLayer">

jsx/views/presentationLayer/ViewProject.js
11: - import Sidebars from './Sidebars'
11: + import Sidebars from '../sidebars/Sidebars'
```

**Comment by davidmfoley** [comment]
oh, interesting. My wild-ass guess here is that trucker can't parse the file via babel (possibly there are some babel plugins that are unknown to it or something) and is falling back to using regex to look for relative paths.  Good catch.

**Comment by davidmfoley** [comment]
Would you mind pasting (or creating a gist) with the contents of Sidebars.js? I read through the code and now I am thinking that this is not a regex parsing issue, but a logic error in how trucker handles the tokenized code while searching for string literals to replace. I think it should be easy enough to fix given an example file. Thanks!

**Comment by eedrah** [comment]
Sure thing. Here's a minimal example (using the create-react-app default babel/webpack config):

```
import React from 'react'
export default () => <a href="/">Test</a>
```

With the command:
```
> trucker frontend/src/jsx/views/{nislSidebars/test.js,}
analyzing 144 files for dependencies
  [====================] 144/144 100% 2.6s elapsed, 0.0s to go

completed analysis

1 changes in 1 files.
frontend/src/jsx/views/nislSidebars/test.js  ->  frontend/src/jsx/views/test.js
2: - export default () => <a href="/">Test</a>
2: + export default () => <a href="./nislSidebars">Test</a>
```

Gives the file (correctly moved to the parent directory):
```
import React from 'react'
export default () => <a href="./nislSidebars">Test</a>
```

**Comment by eedrah** [comment]
(Let me know if you want a real-life example instead of the minimal one)

**Comment by davidmfoley** [comment]
Perfect; thanks.

**Comment by davidmfoley** [comment]
Fixed in 0.7.7; this was due to some pretty lazy code on my part. Thanks for the report and repro.
